examples/ambiguity_detection.rs

```diff
@@ -0,0 +1,154 @@
+//! A test to confirm that `bevy` doesn't have system order ambiguity with DefaultPlugins
+//! This is run in CI to ensure that this doesn't regress again.
+
+use bevy::{
+    ecs::schedule::{LogLevel, ScheduleBuildSettings},
+    prelude::*,
+};
+
+/// A test to confirm that `bevy` doesn't have system order ambiguity with DefaultPlugins
+/// This is run in CI to ensure that this doesn't regress again.
+pub fn main() {
+    let mut app = App::new();
+    app.add_plugins(DefaultPlugins);
+    app.edit_schedule(PreStartup, |schedule| {
+        schedule.set_build_settings(ScheduleBuildSettings {
+            ambiguity_detection: LogLevel::Warn,
+            use_shortnames: false,
+            ..default()
+        });
+    });
+    app.edit_schedule(Startup, |schedule| {
```

Comment:
Is there really no better way to toggle all of these?

Reply:
I'll want to explore that, but ´edit_schedule' wanted a "sized" label, so my first approach with "Box<dyn SheduleLabel>" didn't work out.

There's probably a better way, or worst case a simple macro could be useful.

Reply:
can you get the `Schedules` resource and call `iter_mut` on it?